cluster-autoscaler/cloudprovider/gke/gke_cloud_provider.go

```diff
@@ -121,10 +122,10 @@ func (gke *GkeCloudProvider) GetAvailableMachineTypes() ([]string, error) {
 func (gke *GkeCloudProvider) NewNodeGroup(machineType string, labels map[string]string, systemLabels map[string]string,
 	taints []apiv1.Taint, extraResources map[string]resource.Quantity) (cloudprovider.NodeGroup, error) {
 	nodePoolName := fmt.Sprintf("%s-%s-%d", nodeAutoprovisioningPrefix, machineType, time.Now().Unix())
-	// TODO(aleksandra-malinowska): GkeManager's location will be a region
-	// for regional clusters. We should support regional clusters by looking at
-	// node locations instead.
-	zone := gke.gkeManager.GetLocation()
+	zone, found := systemLabels[kubeletapis.LabelZoneFailureDomain]
```

Comment:
Huh, I just realised this will break all our tests until we start passing zone label. Maybe let's default it for now if it's missing?

Reply:
As discussed offline, this isn't a blocker